openslides/agenda/static/templates/agenda/item-list.html

```diff
@@ -51,7 +51,7 @@ <h1 translate>Agenda</h1>
               class="btn btn-default form-control"
               ng-click="autoNumbering()">
             <i class="fa fa-sort-numeric-asc"></i>
-            <translate>Auto numbering</translate>
+            <translate>Number</translate>
```

Comment:
Hm. Is it easy to understand that the whole agenda will be numberd if one clicks here? Proposal `Number agenda`.